chrome/content/components/wzQuicktextVar.js

```diff
@@ -1,7 +1,9 @@
-Components.utils.import("resource://gre/modules/XPCOMUtils.jsm");
+var EXPORTED_SYMBOLS = ["wzQuicktextVar"];
+
 Components.utils.import("chrome://quicktext/content/utils.js");
 Components.utils.importGlobalProperties(["XMLHttpRequest"]);
 
+var { gQuicktext } = Components.utils.import("chrome://quicktext/content/components/wzQuicktext.js", null);
```

Comment:
There was something about this syntax not working on certain Thunderbird versions. Jörg would know, alternatively realraven or Onno. If you've tested, that's fine as well.

Reply:
Target plattform is only TB60 and there it works. For TB68 I have to drop the null parameter I guess. But this update is not yet the TB68 update but just the minimal changes needed, if we indeed backport the overlay loader to TB60.